src/index.js

```diff
@@ -23,6 +23,7 @@ import wucString from './string';
 import wucTabs from './tabs';
 import wucV6UiSwitch from './v6UiSwitch';
 import wucValidator from './validator';
+import wucServiceStatusAction from './service-status';
```

Comment:
Could be sorted.

Reply:
How ?

I like to sort by file name because how I am supposed to sort

```js
import wucValidator from './validator';
import wucServiceStatusAction from './service-status';
import { stuff, wucStuff } from './stuff';
```

Reply:
It's already sorted in that way.